gradle/native-build.gradle

```diff
@@ -1,9 +1,114 @@
-import groovy.io.FileType
+def dummyFiles   = [ '**/*/impl/Dummy**' ]
+def iosFiles     = [ '**/*/impl/IOS**', '**/**ios.json' ]
+def androidFiles = [ '**/*/impl/Android**', '**/**android.json', '**/dalvik' ]
+def desktopFiles = [ '**/impl/Desktop**', '**/**darwin.json', '**/**linux.json', '**/**windows.json' ]
+
+def projectClasses          = "$project.buildDir/classes/java/main"
+def projectResources        = "$project.buildDir/resources/main"
+def projectIOSResources     = "$project.projectDir/src/ios/resources/"
+def projectAndroidResources = "$project.projectDir/src/android/resources/"
+def projectDesktopResources = "$project.projectDir/src/desktop/resources/"
 
 def sdkPath(String platform) {
     return "/Applications/Xcode.app/Contents/Developer/Platforms/${platform}.platform/Developer/SDKs/${platform}${IOS_VERSION}.sdk";
 }
 
+task androidBuild {
+    group = "native"
+    dependsOn build
+    doLast {
+        def serviceName = getServiceName(project.name)
+        File aar = aarBuild ("$project.buildDir", "$project.projectDir", serviceName, 'android')
+        if (aar == null) {
+            println "Error creating $aar"
+        }
+        nativeBuild("$project.buildDir", "$project.projectDir", serviceName, 'android')
```

Comment:
We have now `ext.nativeBuild` and `task nativeBuild`. Probably better to rename the former to `ext.buildNativeLib`?